src/components/AcceptedValuesField/AcceptedValuesField.test.js

```diff
@@ -23,6 +24,14 @@ const acceptedValuesListProp = [
   },
 ];
 
+const okapiProps = {
+  tenant: 'test tenant',
+  token: 'test token',
+  url: 'test url',
+};
```

Comment:
We use `okapi` prop in a bunch of components, can we create some helper or const like `okapiProp` and use it across tests?